### PR TITLE
Update README.md [Added Electron apps to the list]

### PR DESCRIPTION
**Pull request by parthkrishna1611**
Added lots of more applications to the list, mostly the ones based on Electron.

### CODE REVIEW

**Comment by LewisVo** [comment]
Hi @parthkrishna1611 , thank you so much for your contribution. That is a lot of applications. Please leave me some time to play with them. Meanwhile, could you reorganize these items in alphabetical order?

File: README.md [hunk 1]
```diff
@@ -419,8 +422,8 @@ Brazilian Portugese version : [here](https://github.com/LewisVo/Awesome-Linux-So
 - [Minetest](https://minetest.net) - Open-source Minecraft written in C++ (uses less resources) and includes modding API. ![Open-Source Software][OSS Icon]
 - [Mudlet](mudlet.org) - A cross-platform, open source, and super fast MUD (text-only MMORPGs) client with scripting in Lua. ![Open-Source Software][OSS Icon]
 - [OhMyGiraffe](https://ohmygiraffe.com) - A delightful game of survival. A game about a giraffe eating fruit while being chased by lions.
-- [SuperTuxKart](https://supertuxkart.net) - SuperTuxKart is a 3D open-source arcade racer with a variety characters, tracks, and modes to play. ![Open-Source Software][OSS Icon]
-
+- [Snake Game](https://alpcoskun.com/snake/) - SuperTuxKart is a 3D open-source arcade racer with a variety characters, tracks, and modes to play. [![Open-Source Software][OSS Icon]](https://github.com/alpcoskun/snake)
```
Comment (LewisVo) [inline README.md]: Snake Game's description is wrong.  I am on the way of trying every single app until I notice this. If you are busy and cannot fix this, you can leave it to me.

**Comment by LewisVo** [comment]
Sorry for taking longer than usual

**Comment by parthkrishna1611** [comment]
It's alright. And I'll keep on updating it if I come across something better.

**Pull request merged**